prisma/schema.prisma

```diff
@@ -229,3 +231,28 @@ model DepositHead {
 
   @@map("deposit_head")
 }
+
+model MaspTransaction {
+  id               Int                 @id @default(autoincrement())
+  created_at       DateTime            @default(now()) @db.Timestamp(6)
+  updated_at       DateTime            @default(now()) @updatedAt @db.Timestamp(6)
+  transaction_hash String              @db.VarChar
+  block_hash       String              @db.VarChar
+  asset            String              @db.VarChar
+  type             MaspTransactionType
+  block_sequence   Int
+  network_version  Int
+  main             Boolean
+  amount           Int
+  event            Event?
+
+  eventId Int?
```

Comment:
```suggestion
  event_id Int?
```

Reply:
Will these be unique?

Reply:
I am not sure for these two questions, I was imagining the event represents a sent event from client. Not sure if we want to dedupe and prevent secondary save or if we want to save both and have one to many for masp <- event. I think you guys would be better at making the decision on that given you have been through a couple testnet phases already